### DISCUSSION TITLE
Bonus points winter term 2024/25

### DISCUSSION THREAD

**Discussion by pkeilbach** [pkeilbach/htwg-practical-nlp#204] [General]
Here is an overview of the bonus point for our semester. Please let me know if something is wrong!

| Student          | Bonus Points Assignments | Bonus Points Contributions | PyPI Package | Total  |
|------------------|--------------------------|----------------------------|--------------| ------ |
| @Alex-J-W        | 🏅🏅🏅🏅🏅             | 🏅🏅🏅🏅                 | 🏅🏅       |  10 (11)|
| @KrasserKai      | 🏅🏅🏅🏅🏅             | 🏅🏅🏅🏅🏅              |              | 10     |
| @TobiTgl         | 🏅🏅🏅🏅🏅             | 🏅🏅🏅🏅🏅              |              | 10     |
| @BeneHTWG        | 🏅🏅🏅🏅🏅             | 🏅🏅🏅🏅🏅🏅🏅         |              | 10 (12)|
| @SponsoredByPuma | 🏅🏅🏅🏅🏅             | 🏅🏅🏅🏅🏅🏅            |             | 10 (11)|
| @lauraschrwr     | 🏅🏅🏅🏅🏅             | 🏅                        |              | 6      |
| @TimSchober      | 🏅🏅🏅🏅🏅             |                            |              | 5      |
| @ari21itwm       | 🏅🏅🏅🏅🏅             |                            |              | 5      |
| @mayah1010       | 🏅🏅🏅🏅🏅             |                            |              | 5      |
| @julss97         | 🏅🏅🏅🏅🏅             |                            |              | 5      |
| @khadidja-kebaili| 🏅🏅🏅🏅🏅             |                            | 🏅🏅        | 7      |

**Comment by TobiTgl** [comment]
Hi @pkeilbach, habe gerade nochmal meine PRs bzgl. den Bonuspunkten gecheckt, wenn ich es richtig sehe fehlen hier zwei Punkte in der Übersicht, müsste insgesamt auch auf 10 kommen :)

**Reply by pkeilbach** [reply]
Stimmt, danke!
